CHANGELOG.markdown

```diff
@@ -28,12 +28,24 @@
     the `PolyKinds` extension at use sites in order to typecheck.
   * Because TH-generated optics now quantify more kind variables than they did
     previously, this can affect the order of visible type applications.
+
```

Comment:
Was this whitespace-only change intended?

Reply:
conflict resolution artficact. fixed.